src/models/UsersModel.js

```diff
@@ -0,0 +1,51 @@
+class UsersModel {
+    constructor({
+      name,
+      email,
+      profilePicture = 'http://localhost:8000/images/userProfileImage.jpg', // Default image URL
```

Comment:
This is not a valid image URL when it gets sent to a frontend. We should either use string URLs that point to places on the internet (i.e. https://encrypted-tbn1.gstatic.com/images?q=tbn:ANd9GcT0MF20H5ZhHzeW3VMVOoN07oVWHMDUnEpOb6LY5ExNy0C_9dOjCXJVwTJPZtKpF77CBn_aN3MKxz4j9XL9riu5-I77IoINehPXDmhpgw) or find a way to let users upload images and store them in Firebase. The latter sounds possibly like it would cost money and is more complexity than I think we should have right now.

Reply:
I will change the path of image URL For now I have added null value for the profilePicture field.